public/wp-content/themes/glific/src/scss/style.scss

```diff
@@ -62,3 +62,4 @@ body{
 @import 'media/xl';
 @import 'media/hd';
 @import 'header/style';
+@import 'page/getStarted.scss';
```

Comment:
Please check with @mohitgusain if we are using camel casing for scss file names.